app/views/fileChecksResults.scala.html

```diff
@@ -6,17 +6,15 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         @progressIndicator(Messages("fileChecksResults.progress"))
-
-        <div class="govuk-panel govuk-panel--confirmation">
-            <h1 class="govuk-panel__title">
-                Checks Complete
-            </h1>
-            <div class="govuk-panel__body">
-                @if(consignmentInfo.numberOfFiles > 1) {
-                    @Messages("fileChecksResults.description.plural", consignmentInfo.parentFolder, consignmentInfo.numberOfFiles)
-                } else {
-                    @Messages("fileChecksResults.description.singular", consignmentInfo.parentFolder, consignmentInfo.numberOfFiles)
-                }
+        <h1>Results of your checks</h1>
```

Comment:
Could you add class="govuk-heading-l" to the h1 tag, please?

Reply:
I've added the class to the h1 tag.